tests/generics/tuninstantiated_failure.nim

```diff
@@ -11,6 +11,6 @@ func `[]`[T, K](x: var Test[T, K], idx: int): var Test[T, K] =
   x
 
 var b: Something
-# Should give a type-mismatch since Something isn't a valid Test
+# Should give an error since Something isn't a valid Test
 b[0].name = "Test" #[tt.Error
-            ^  type mismatch]#
+ ^  expression '' has no type (or is ambiguous)]#
```

Comment:
But that is a worse error message. Maybe we can wrap the `n` in an `nkError` node and use it inside of this error message?

Reply:
I don't know. I'm just packporting the fix https://github.com/nim-lang/Nim/commit/642ac0c1c31063ae966d8448c64c539203432e94, which was made by @ringabout several months ago. We were getting a crash in nimsuggest on our code base, and I did a git bisect to find where the bug was fixed, but I didn't make the fix. Is there a follow-up commit, where your suggestion was implemented, or are you requesting that we change @ringabout's fix?

Reply:
Didn't realize it was a backport.